docs/team/ldinghan.md

```diff
@@ -45,9 +46,11 @@ Given below are my contributions to the project:
   * User Guide:
     * Added documentation for the Session Management Section.
     * Designed the annotated images displaying the different sections of the UI of the product.
-
+    * Reorganised the order of sections in a more logical order.
   * Developer Guide:
     * Designed the class diagram for `Session`.
+    * Designed the sequence diagram for `TakeAttendanceCommand`.
```

Comment:
perhaps could just include the diagram inside here for easy reference

Reply:
sounds like a good idea!